core/include/emulate.h

```diff
@@ -142,6 +140,7 @@ typedef struct em_operand_t {
         } mem;
         struct operand_reg_t {
             uint32_t index;
+            uint32_t shift;
```

Comment:
nit: Be consistent with the type of `shift`. `gpr_read_shifted()` uses `size_t`.

Reply:
Instead, I've modified the previous function definitions to make their `shift` argument `uint32_t`.

Reply:
OK, that works too.